packages/bits/spec/components/table/table.visual.ts

```diff
@@ -80,6 +83,12 @@ describe(`Visual tests: ${name}`, () => {
         await camera.say.cheese("Edit columns");
         await expanders.customActions.click();
 
+        await expanders.selectPinnedHeader.click();
+        await selectPinnedHeaderTable.getCell(1, 0).click();
+        await browser.executeScript("document.getElementById('table-select-pinned-header').getElementsByClassName('nui-table__container')[0].scrollTop = '20'");
```

Comment:
we may want to have this in a bit generic format in our Helpers, so we could reuse it. Something like scrollUp() or scrollDown()

Reply:
I think it is not a generic script, which we can reuse in our Helpers